config/WordPress.debug.xcconfig

```diff
@@ -1,4 +1,4 @@
-#include "Version.internal.xcconfig"
+#include "Version.public.xcconfig"
```

Comment:
We weren't really using the unique internal version xcconfig build number values once the release cycle builds were disabled in March. This change will just have the debug builds use whatever the build number is in the public version file.